app/dl_formats.py

```diff
@@ -23,7 +23,7 @@ def get_format(format: str, quality: str) -> str:
         # Quality is irrelevant in this case since we skip the download
         return "bestaudio/best"
 
-    if format in ("m4a", "mp3"):
+    if format in ("m4a", "mp3", "opus", "wav"):
```

Comment:
Since there's a lot of repetition of this quartet, perhaps it's worth creating a constant `AUDIO_FORMATS` or something like that, and using it in all those `if`s